e2e/cypress/integration/team/teams_spec.js

```diff
@@ -182,4 +182,50 @@ describe('Teams Suite', () => {
         // * Ensure user is logged out
         cy.url({timeout: TIMEOUTS.LARGE}).should('include', 'login');
     });
+
+    it('TS14632 Team Icon should be Removed', () => {
+        cy.apiLogin('sysadmin');
+        cy.apiCreateTeam('test-team', 'Test Team').then(() => {
+            cy.visit('/');
+        });
+        cy.get('#sidebarHeaderDropdownButton').click();
+        cy.get('#teamSettings').click();
+
+        // 1. On a Team taht has a team icon
+        cy.get('#team_iconEdit').click();
```

Comment:
nit: Above 3 steps are called in different places.  You might want to extract those into a separate function.